tests/production/DeleteUserSettingsFromCouchTests.js

```diff
@@ -0,0 +1,214 @@
+/**
+Tests for deleting  test gpii keys and 'user' prefs safe.
+
+Copyright 2019 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii");
+
+fluid.require("%gpii-universal");
+
+gpii.loadTestingSupport();
+
+fluid.registerNamespace("gpii.tests.productionConfigTesting");
+
+require("./Common.js");
+
+fluid.defaults("gpii.tests.productionConfigTesting.deleteUserSettings", {
+    gradeNames: ["fluid.test.sequenceElement"],
+    sequence: [
+        { funcName: "fluid.log", args: ["Delete 'user' test prefs safes tests:"]},
+        {
+            func: "{getSettingsUserKey}.send",
+            args: [null, { port: "5984" }]
+        }, {
+            event: "{getSettingsUserKey}.events.onComplete",
+            listener: "gpii.tests.productionConfigTesting.testGetForDeletion"
+        }, {
+            func: "{getSettingsUserPrefsSafe}.send",
+            args: [null, { port: "5984" }]
+        }, {
+            event: "{getSettingsUserPrefsSafe}.events.onComplete",
+            listener: "gpii.tests.productionConfigTesting.testGetForDeletion"
+        }, {
+            func: "{getGpiiKeyNoPrefsSafe}.send",
+            args: [null, { port: "5984" }]
+        }, {
+            event: "{getGpiiKeyNoPrefsSafe}.events.onComplete",
+            listener: "gpii.tests.productionConfigTesting.testGetForDeletion"
+        }, {
+            funcName: "gpii.tests.productionConfigTesting.sendPrefsSafeId",
```

Comment:
Can this function be renamed to something like "getPrefsSafeDoc"? These new names are just suggestions from my understanding of what they do. Feel free if you come up with better names. Thanks.

Reply:
Yes, `getPrefsSafeDoc()` is better.